packages/react-storage/src/components/StorageBrowser/displayText/types.ts

```diff
@@ -39,6 +39,8 @@ export interface DefaultLocationsViewDisplayText
   tableColumnFolderHeader: string;
   tableColumnBucketHeader: string;
   tableColumnPermissionsHeader: string;
+  tableColumnActionsHeader: string;
+  downloadLabel: (fileName: string) => string;
```

Comment:
Prefix `displayText` callbacks with `get`:
```suggestion
  getDownloadLabel: (fileName: string) => string;
```